org.gnome.Contacts.json

```diff
@@ -0,0 +1,215 @@
+{
+    "app-id": "org.gnome.Contacts",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "gnome-contacts",
+    "tags": [ "stable" ],
+    "rename-icon": "gnome-contacts",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.gnome.ControlCenter",
+        "--talk-name=org.gnome.OnlineAccounts",
+        "--talk-name=org.gnome.evolution.dataserver.AddressBook9",
+        "--talk-name=org.gnome.evolution.dataserver.Sources5",
+        "--talk-name=org.gnome.evolution.dataserver.Subprocess.Backend.*",
+        "--talk-name=org.freedesktop.Telepathy.AccountManager",
+        "--filesystem=home:ro",
+        "--filesystem=xdg-data/pixmaps/faces:ro",
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
```

Comment:
@TingPing: question: are these flags actually useful for anything here for example?

Reply:
Some of the autotools systems *may* need them, its hard to say without reading build logs.